### PR TITLE
kubectl-gs login: update docs for --keep-context and --self-contained flags

### PR DESCRIPTION
**Pull request by kuosandys**
Update for kubectl-gs `login` documentation for the changes: https://github.com/giantswarm/kubectl-gs/pull/660 and https://github.com/giantswarm/kubectl-gs/pull/661.

### CODE REVIEW

**Review by anvddriesch — APPROVED** [review]

File: src/content/ui-api/kubectl-gs/login.md
Comment (marians) [inline src/content/ui-api/kubectl-gs/login.md:59]: This is placed in the `**management cluster**` section. Is that correct? Is the flag only applicable with MC login, not WC?

Comment (kuosandys) [reply src/content/ui-api/kubectl-gs/login.md:59]: Yes it's only applicable with MC login at the moment, but I just saw that there are some changes in https://github.com/giantswarm/kubectl-gs/pull/661/files that will make it available for WC as well.

I think maybe this PR is premature - we can also align the docs after the other PR is also merged?

Comment (marians) [reply src/content/ui-api/kubectl-gs/login.md:59]: Yes, we have to wait for a release anyway.

**Review by marians — APPROVED** [review]

**Pull request merged**
